### DISCUSSION TITLE
MACRO DOESN'T START NO MORE

### DISCUSSION THREAD

**Discussion by polin79** [WarmUpTill/SceneSwitcher#1252] [Q&A] [answered]
I have OBS studio for Linux (last release from Flathub) with Advanced Scene Switcher version 1.28.0. In the last few days I have tried to manually run a macro where in the different actions there was also "Start streaming". After the test I have manually stopped the streaming even if the macro was still running and now the macro doesn't start no more and I think the problem could be that the macro has not been manually stopped. Is there a way to stop in advance a running macro? Could you kindly help me how to solve problems that macros don't start no more?

Thanks
Paolo

**Comment by WarmUpTill** [comment]
Unfortunately without more information it is very difficult to say what the problem might be.

Can you please [share](https://github.com/WarmUpTill/SceneSwitcher/wiki/Exporting-and-importing-individual-macros) the macro, which is not working for you?
It might also help to get a log file of the issue with [verbose logging](https://github.com/WarmUpTill/SceneSwitcher/wiki/Troubleshooting#logging) enabled to get more information what the state of each macro and its conditions is.

**Reply by polin79** [reply]
Hi, thanks for replying. In attachment the 7 macros which are no more working. I have activated the verbose logging, but how to export the log?
[Macro7.txt](https://github.com/user-attachments/files/17579309/Macro7.txt)
[Macro6.txt](https://github.com/user-attachments/files/17579310/Macro6.txt)
[Macro5.txt](https://github.com/user-attachments/files/17579311/Macro5.txt)
[Macro4.txt](https://github.com/user-attachments/files/17579313/Macro4.txt)
[Macro3.txt](https://github.com/user-attachments/files/17579315/Macro3.txt)
[Macro2.txt](https://github.com/user-attachments/files/17579316/Macro2.txt)
[Macro1.txt](https://github.com/user-attachments/files/17579317/Macro1.txt)

**Reply by polin79** [reply]
I tried to change the macro starting of the Macro3.txt with the start a few minutes ago and the macro this time has started without problems, but sometimes macros don't start, mainly if I have manually run macros and manually stopped live streaming. In attachment the modified Macro3.txt. Which is the right way to stop in advance macro running?
I tried to export log, but I don't know if it's the right way.

Thanks
Paolo

[Macro3_V2.txt](https://github.com/user-attachments/files/17579356/Macro3_V2.txt)
[adv-ss-Streaming MEDJU.json](https://github.com/user-attachments/files/17579391/adv-ss-Streaming.MEDJU.json)

**Reply by WarmUpTill** [reply, marked as answer]
>Which is the right way to stop in advance macro running?

Usually you simply stop the plugin.
Otherwise you can also set up a helper macro with an action like this if you want to stop ony one specific macro:
![image](https://github.com/user-attachments/assets/94bdae69-34ff-410d-b834-6e7bd1bd6512)

>I tried to export log, but I don't know if it's the right way.

I was referring to an OBS log, which you can find under Help > Log Files.

Just a general tip:
For your particular macros you might want to tick this option in each of the macro's settings:
![Example](https://github.com/user-attachments/assets/06fecfc1-03c0-490d-9c60-aa1f1f10dd1e)

This will result in the macro to be stopped before you try to rerun it.

Also I would highly recommend to **_disable_** [this setting](https://github.com/WarmUpTill/SceneSwitcher/wiki/General-tab-overview#match-behavior) on the General tab, which you might have enabled by accident:
![image](https://github.com/user-attachments/assets/0d9adf76-e22b-4837-bbaa-b7c398548840)

**Reply by polin79** [reply]
Thanks a lot for all suggestions. In attachment logs of this evening. Tomorrow after the live streaming I will send other logs
[2024-10-30 22-23-00.txt](https://github.com/user-attachments/files/17579753/2024-10-30.22-23-00.txt)

**Reply by polin79** [reply]
Now everything seems to work properly. Thank you very very much for the support
